packages/remix/src/ssr/types.ts

```diff
@@ -10,6 +10,7 @@ export type RootAuthLoaderOptions = {
   frontendApi?: string;
   publishableKey?: string;
   jwtKey?: string;
+  proxyUrl?: string;
```

Comment:
❓Doesn't this expose the proxyUrl to user when calling `getAuth` & `rootAuthLoader` and passing options ?

Reply:
@panteliselef I omitted it in `rootAuthLoader`. In `getAuth` we pick certain values so we are ok